src/api-reference/request/get-started.markdown

```diff
@@ -0,0 +1,173 @@
+---
+title: Get Started
+layout: reference
+---
+
+# Request
+Concur Request automates the spend request and approval process for both travel and everyday expenses, giving you the data you need to accurately track and better control spending. By increasing visibility into planned expenses and up-to-date budget data, you can make strategic spending decisions before any spending actually occurs. The Request resource provides many abilities, including requests creation and transition into the approval workflow.
+
+## Get Started
+
+- [Overview](#overview)
+- [Version](#version)
+  - [Overview version V4.0](#overviewV4)
+- [Regional Availability](#regional)
+  - [With Swagger](#swagger)
+  - [With Curl](#curl)
+- [Explore the API](#explore-api)
+  - [Prerequisites](#prerequisites)
+  - [Acquire an Access Token](#token)
+  - [Explore the API in JavaScript](#javascript)
+    - [Retrieve a User Access Token](#retrieve-token)
+    - [Post a Request](#post-request)
+
+
+### <a name="overview"></a>Overview
+
+The Request V4 API has 4 resources:
+
+1. __Expense resource__ - You can  select, create, update or delete an expense or show the list of expense link to a request.
+2. __Request resource__ - You can select, create, update or delete a request or show a list of requests.
+3. __Travel Agency resource__ - You can get the description of a travel agency office.
+4. __Workflow resource__ - You can advance a request in a new authorized status.
+
+Manage documents used for pre-spend authorizations within Concur Request.
+
+### <a name="version"></a>Version
+
+- 4.0
+- [3.0](/api-reference-deprecated/version-three/request/request.html)
+- [1.0](/api-reference-deprecated/version-one/Travel/travel-request.html)
+
+#### <a name="overviewV4"></a>Overview of Version 4.0
+
+Version 4.0 of the Requests API offers features like passive approval, Data retention, rigth to be forgotten.
```

Comment:
Version 4.0 of the Request API 
_(no s on Request)_
_note: do we consider data retention and right to be forgotten as features?_

Reply:
Removed : this is not publicly exposed